CONTRIBUTING.md

```diff
@@ -99,6 +100,35 @@ Our different types of reviews:
   3. **Only comments.** You must address all the comments and need another review until you merge.
   4. **Request changes.** Only use if something critical is in the PR that absolutely must be addressed. We usually use `h` comments for that. When someone requests changes, the same person must approve the changes to allow merging. Use this sparingly.
 
+## Commit, Issue & PR guidelines
+
+### Commits
+
+For commit messages, we use the format:
+
+```
+<type>(<scope>): <subject> (<github-id>)
+````
+
+For example: `feat(core): Set custom transaction source for event processors (#5722)`.
+
+See [commit message format](https://develop.sentry.dev/commit-messages/#commit-message-format) for details.
+
+### Issues
+
+Issues should at least be categorized by package, for example `package: Node`.
+Additional labels for categorization can be added, and the Sentry SDK team may also add further labels as needed.
+
+### Pull Requests (PRs)
+
+PRs are merged via `Squash and merge`.
+This means that all commits on the branch will be squashed into a single commit, and commited as such onto master.
+
+* The PR name can generally follow the commit name (e.g. `feat(core): Set custom transaction source for event processors`)
+* Use labels the same way as for issues
```

Comment:
I don't think we should recommend this, folks normally don't label their PRs (what we maybe use is labeling by package, but that is normally only done for external contributor PRs)

Reply:
👍 will just remove this then!